Monitoring in a Box/Ansible/roles/mib_setup/tasks/set_up_mib_service.yaml

```diff
@@ -19,6 +19,7 @@
     - config
     - db
 
-- name: Start MiB service
-  docker_compose:
-    project_src: /opt/MiB/   
+- name: Pull required MiB service docker images
+  shell: docker-compose pull
+  args:
+    chdir: /opt/MiB/
```

Comment:
Will this work?
```suggestion
docker-compose:
  project_src: /opt/MiB/
  pull: yes
  state: absent
```

Reply:
I have try this solution before, but it would not work. Because by putting the absent, the pull command would not run also, since container is not required to run. But i will try it again to confirm.

Reply:
I tried again, and it does not pull the image when the state is absent.